File: alx-low_level_programming/0x01-variables_if_else_while/1-last_digit.c
#include <stdlib.h>
#include <time.h>
#include <stdio.h>
/**
 * main - Entry point
 * Return: 0 (after compiltion)
 */
/* Task zero*/
int main(void)
{
	int n;
	int lastDigit;

	srand(time(0));

	n = rand() - RAND_MAX / 2;
	lastDigit = n % 10;
	if (lastDigit > 5)
	{
		printf("Last digit of %d is %i and is greater than 5\n", n, lastDigit);
	}
	else if (lastDigit == 0)
	{
		printf("Last digit of %d is %i and is 0\n", n, lastDigit);
	}
	else if (lastDigit < 6 && lastDigit != 0)
	{
		printf("Last digit of %d is %i and is less than 6 and not 0\n", n,
lastDigit);
	}
	return (0);
}

File: alx-higher_level_programming/0x01-python-if_else_loops_functions/1-last_digit.py
#!/usr/bin/python3
import random
number = random.randint(-10000, 10000)
if number < 0:
    val = number % -10
else:
    val = number % 10
if val > 5:
    print(f"Last digit of {number:d} is {val:d} and is greater than 5")
elif val == 0:
    print(f"Last digit of {number:d} is {val:d} and is 0")
elif val < 6 and val != 0:
    print(f"Last digit of {number:d} is {val:d} and is less than 6 and not 0")
